package.nls.json

```
@@ -430,6 +433,8 @@
     "AWS.s3.validateBucketName.error.invalidStart": "Bucket name must start with a lowercase letter or number",
     "AWS.s3.validateBucketName.error.misusedPeriods": "Periods in bucket name must be surrounded by a lowercase letter or number",
     "AWS.s3.validateBucketName.error.resemblesIpAddress": "Bucket name must not resemble an IP address",
+    "AWS.s3.validateTime.error.notNumber": "Input has to be a number",
```

Redundant with the string under it. Just test both at the same time and use the same string.

---

Avoid colloquial language

```suggestion
    "AWS.s3.validateTime.error.notNumber": "Input must be a number",
```